tests/config.py

```diff
@@ -3,6 +3,16 @@
 CLIENT_DEPLOYMENT_FILE = './k8s/client-w-conf.yml'
 CLIENT_STATEFULSET_FILE = './k8s/client-w-conf-ss.yml'
 CURL_POD_FILE = './k8s/curl.yml'
+ELASTIC_CONF_DIR = './elk/elastic/'
+FILEBEAT_CONF_DIR = './elk/filebeat/'
+FLUENT_BIT_CONF_DIR = './elk/fluent-bit/'
+KIBANA_CONF_DIR = './elk/kibana/'
+LOGSTASH_CONF_DIR = './elk/logstash/'
+
+ES_USER_LOCAL = "elastic"
+ES_PASS_LOCAL = "gavradon"
```

Comment:
it's better not to include the password here as plain text

Reply:
is base64 adequate?

Reply:
no, we should not store passwords in text files. It should be a GitHub secret attached to the repository and passed in through the environment.